packages/cli/src/lib/cmds/build.ts

```diff
@@ -143,6 +159,20 @@ class Build {
     const manifestFile = this.args.manifest || path.join(process.cwd(), TWA_MANIFEST_FILE_NAME);
     const twaManifest = await TwaManifest.fromFile(manifestFile);
 
+    const manifestCheck = await this.checkManifest(manifestFile);
```

Comment:
Similarly, could this variable be `hasManifestChanged`?

Reply:
done.